### PR TITLE
SZVAV for other equipment

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
Please change this line to a description of the pull request, with useful supporting information including whether it is a new feature, or fixes a defect, a cross reference to any defects addressed in this PR, the type of changes to be made, and whether diffs are expected/justified based on this change.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@rraustad @lgentile it has been 16 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@rraustad @lgentile it has been 26 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
@rraustad You planning on wrapping this up in time for 8.7 freeze?

**Comment by rraustad** [comment]
You mean...Do I want to get paid? Yes.

szvav_progress

File: design/FY2017/Final NFP-Single-Zone VAV Fan Control for Other Equipment Types.md [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+add new key here for capacity control method:
+
+       \key ASHRAE90VariableFan
+
+with description:
+
+       \default Load
+       \note Load control requires a Controlling Zone name.
+       \note SetPoint control requires set points at coil outlet node.
+       \note ASHRAE90VariableFan control requires a Controlling Zone name, water coils and two-speed or variable-speed fan
```
Comment (mjwitte) [inline design/FY2017/Final NFP-Single-Zone VAV Fan Control for Other Equipment Types.md:221]: @rraustad So, why won't this work just the same with DX coils?

Comment (rraustad) [reply design/FY2017/Final NFP-Single-Zone VAV Fan Control for Other Equipment Types.md:221]: Thanks for reviewing. I just got single-speed DX cooling and heating coils working. I'll update that note. I haven't tested multi-speed coils yet nor know if I have enough time to finish those coils (approaching 200 hr budget). Will still allow any coils in UnitarySystem but only apply the ASHRAE 90.1 methodology to coils I have tested. I've also revised these notes so some will change.

**Comment by nrel-bot-3** [comment]
@rraustad @lgentile it has been 14 days since this pull request was last updated.

**Comment by mjwitte** [comment]
@rraustad It looks like this PR has been replaced with #5994? If so, please add this NFP to #5994 and close this PR.

**Comment by rraustad** [comment]
Merged with 5994 and closed.

**Pull request closed without merging**